code/modules/mob/living/carbon/metroid/life.dm

```diff
@@ -347,9 +347,8 @@
 
 					if(istype(L, /mob/living/carbon/human) && dna) //Ignore slime(wo)men

```

Comment:
This should probably check for if(istype(L, /mob/living/carbon/human) && **species**)